Header header

# угловая скорость по курсу
float64 rate_heading

# угловая скорость по крену
float64 rate_roll

# угловая скорость по дифференту
float64 rate_pitch